wp-content/themes/csisnuclearnetwork/template-parts/newsletter-block-acf.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * The about PONI section of the home page,
+ * including information about directors and a team photo
+ *
+ * @link https://developer.wordpress.org/themes/basics/template-hierarchy/
+ *
+ * @package CSIS iLab
+ * @subpackage @package nuclearnetwork
+ * @since 1.0.0
+ */
+
+?>
+
+<?php 
+    if( ! class_exists('ACF') ) {
+    return;
+    }
+
+    $news_args = array(
+    'numberposts' => 1,
+    'post_type'   => 'news',
+    'fields' => 'ids'
+    );
+    
+    $latest_news = get_posts( $news_args );
+
+    if( $latest_news ) {
+        $recentNewsUrl = get_permalink($latest_news[0]);
+    }
+
+    $dailyNewsletterName = get_field('daily_newsletter_name');
+    $dailyNewsletterDesc = get_field('daily_newsletter_description');
+    $dailyNewsletterLinks = "<div class='recent-news'><a href=" . $recentNewsUrl . ">Most recent newsletter" .nuclearnetwork_get_svg('chevron-right') . "</a><a href=" . get_post_type_archive_link( 'news' ) . ">All Nuclear Policy News" .nuclearnetwork_get_svg('chevron-right') . "</a></div>";
+    $monthlyNewsletterName = get_field('monthly_newsletter_name');
+    $monthlyNewsletterDesc = get_field('monthly_newsletter_description');
+    $monthlyNewsletterLinks = "<div class='recent-news'><a href=" . $recentNewsUrl . ">Most recent newsletter" .nuclearnetwork_get_svg('chevron-right') . "</a></div>";
```

Comment:
This link should be different than the daily news link. Did you follow up with Christina to ask her where it was supposed to link to?

Reply:
She said the same place